core/src/elements/ons-ripple/index.js

```diff
@@ -0,0 +1,220 @@
+/*
+Copyright 2013-2015 ASIAL CORPORATION
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import util from 'ons/util';
+import BaseElement from 'ons/base-element';
+import GestureDetector from 'ons/gesture-detector';
+import AnimatorJS from './animator-js';
+import AnimatorCSS from './animator-css';
```

Comment:
The element used to have two animator libraries which it could use, depending on what is preferred. The API is the same for both. Nevertheless I commented out the one not being used right now.